app/views/shared/_notification-bar.html.erb

```diff
@@ -0,0 +1,62 @@
+ <div class="col-md-4">
+    <div class="panel panel-info">
+        <div class="panel-heading">
+            <i class="fa fa-bell fa-fw"></i>Notifications Panel
+        </div>
+
+        <div class="panel-body">
+            <div class="list-group">
+
+                <a href="#" class="list-group-item">
+                    <i class="fa fa-twitter fa-fw"></i>3 New Followers
+                <span class="pull-right text-muted small"><em>12 minutes ago</em>
+                </span>
+                </a>
+                <a href="#" class="list-group-item">
+                    <i class="fa fa-envelope fa-fw"></i>Message Sent
+                <span class="pull-right text-muted small"><em>27 minutes ago</em>
+                </span>
+                </a>
+                <a href="#" class="list-group-item">
+                    <i class="fa fa-tasks fa-fw"></i>New Task
+                <span class="pull-right text-muted small"><em>43 minutes ago</em>
+                </span>
+                </a>
+                <a href="#" class="list-group-item">
+                    <i class="fa fa-upload fa-fw"></i>Server Rebooted
+                <span class="pull-right text-muted small"><em>11:32 AM</em>
+                </span>
+                </a>
+                <a href="#" class="list-group-item">
+                    <i class="fa fa-bolt fa-fw"></i>Server Crashed!
+                <span class="pull-right text-muted small"><em>11:13 AM</em>
+                </span>
+                </a>
+                <a href="#" class="list-group-item">
+                    <i class="fa fa-warning fa-fw"></i>Server Not Responding
+                <span class="pull-right text-muted small"><em>10:57 AM</em>
+                </span>
+                </a>
+
+                <a href="#" class="list-group-item">
+                    <i class="fa fa-bolt fa-fw"></i>Server Crashed!
+                <span class="pull-right text-muted small"><em>11:13 AM</em>
+                </span>
+                </a>
+                <a href="#" class="list-group-item">
+                    <i class="fa fa-warning fa-fw"></i>Server Not Responding
+                <span class="pull-right text-muted small"><em>10:57 AM</em>
+                </span>
+                </a>
+                <a href="#" class="list-group-item">
+                    <i class="fa fa-shopping-cart fa-fw"></i>New Order Placed
+                <span class="pull-right text-muted small"><em>9:49 AM</em>
+                </span>
+                </a>
+            </div>
+            <!-- /.list-group -->
+            <a href="#" class="btn btn-info btn-block">View All Alerts</a>
+        </div>
+
+    </div>
+</div>
```

Comment:
add new line here as well.
is it really required here.
if it is as this is layout and not dynamic file. why are adding all this hard coded data here.

Reply:
it's required to show some important statistics here. it will be static just for the time being when implementation is done there will be no dummy data.